content/en/docs/instrumentation/java/automatic/server-config.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Application server configuration
```

Comment:
This currently shows at the top level (in the demo version).  How do we make sure it shows up in `instrumentation/java/automatic/`?  Maybe this is a bug with the demo?

Reply:
Oh, I think the files moved to /languages, could it be? Checking.

Reply:
@tylerbenson I've moved the files, check the demo: https://deploy-preview-3459--opentelemetry.netlify.app/docs/languages/java/automatic/server-config/

Reply:
looks much better, thanks!